packages/react-vapor/src/components/table-hoc/utils/TableHOCUtils.ts

```diff
@@ -88,13 +88,18 @@ const getTablePredicates = (tableId: string, state: IReactVaporState): ITableHOC
 
 const getDatePickerId = (tableId: string) => `${tableId}-date-range`;
 
+const redirectToUrl = (link: string) => {

```

Comment:
This should rather go in the [`UrlUtils`](https://github.com/coveo/react-vapor/blob/fcd398a8b29252804be8d569b0d05f9655c97220/packages/react-vapor/src/utils/UrlUtils.ts)

Reply:
will move it thx !